build/release.mk

```diff
@@ -43,7 +43,7 @@ release/fix-archive:
 .PHONY : release/sign/nix
 release/sign/nix:
 	@echo "===> $(INTEGRATION) === [release/sign] signing packages"
-	@bash $(CURDIR)/build/nix/sign.sh
+	@bash  /usr/local/bin/sign.sh
```

Comment:
is this executed inside the build machine [here](https://github.com/newrelic/coreint-automation/pull/105/files)?

Reply:
yes